query/reverse.js

```diff
@@ -65,13 +65,24 @@ function generateQuery( clean ){
   // where point.lan/point.lon are provided in the
   // absense of boundary.circle.lat/boundary.circle.lon
   if( check.number(clean['boundary.circle.lat']) &&
-      check.number(clean['boundary.circle.lon']) &&
-      check.number(clean['boundary.circle.radius']) ){
-    vs.set({
-      'boundary:circle:lat': clean['boundary.circle.lat'],
-      'boundary:circle:lon': clean['boundary.circle.lon'],
-      'boundary:circle:radius': clean['boundary.circle.radius'] + 'km'
-    });
+      check.number(clean['boundary.circle.lon']) ){
+
+        vs.set({
+          'boundary:circle:lat': clean['boundary.circle.lat'],
+          'boundary:circle:lon': clean['boundary.circle.lon']
+        });
+
+        if (check.undefined(clean['boundary.circle.radius'])){
+          // for coarse reverse when boundary circle radius is undefined
```

Comment:
Boundary.circle.lat and boundary.circle.lon are [currently unsupported](http://pelias.github.io/compare/#/v1/reverse%3Fboundary.circle.lat=40&boundary.circle.lon=-74&point.lat=40.1&point.lon=-74.1) anyway. There shouldn't be a non-coarse reverse query that has the point coordinates without a radius.